### DISCUSSION TITLE
Gradient tittle not rendering well on iOS phone

### DISCUSSION THREAD

**Discussion by mllamazares** [tailwindlabs/tailwindcss#5474] [Help] [answered]
### What version of Tailwind CSS are you using?

2.2.2

### What build tool (or framework if it abstracts the build tool) are you using?

next.js 

### What version of Node.js are you using?

14.0.0

### What browser are you using?

Chrome

### What operating system are you using?

Debian 10

### Reproduction repository

https://github.com/mllamazares/mllamazares.github.io/

### Describe your issue

This is the beta version of my blog using tailwind.
https://mllamazares.github.io/

When accessing via iOS phones it does not render the second word ("Analyst") well:
![image](https://user-images.githubusercontent.com/5461023/132092012-4e6f4d06-5ffe-43ba-8244-d6f3dd139108.png)

In Android phones or via browser, it's displayed as expected:
![image](https://user-images.githubusercontent.com/5461023/132092055-037cc5c3-d9af-4b22-9344-3d7f890690b4.png)

This is the code to generate that gradient:
```
          <h1 className="pb-2 inline font-extrabold leading-snug tracking-tight text-title text-4xl md:text-maintitle md:leading-snug">
            Hello, my name's Miguel. I'm a passionate{' '}
            <p className="inline text-transparent bg-clip-text bg-gradient-to-br from-primary-300 to-primary-700">
              {' '}
              Systems Analyst
            </p>{' '}
            based in northen Spain.
          </h1>
```
Here you can check [the rest of the template](https://github.com/mllamazares/mllamazares.github.io/blob/master/pages/index.js) for further context. 

Any idea of what's going on here?

**Comment by adamwathan** [comment, marked as answer]
Hey! Going to convert this to a discussion as it's a more general CSS question and not a bug in our build tool or anything (you'll see the same issue writing raw CSS in iOS Safari).

This might help: https://discourse.webflow.com/t/multi-line-gradient-text-safari-ios-fix-i-e-prevent-disappearing-text/135029

**Reply by mllamazares** [reply]
Thanks! It worked like a charm! 👌

BTW, don't you think it would be interesting to add this fix by default? (despite it is not strictly tailwind-related). Something like:

```css
.bg-clip-text {
  -webkit-box-decoration-break: clone;
}
```

**Reply by simonswiss** [reply]
Hey!

`clone` is not _always_ the intended outcome here, so it's better to compose it with the existing `box-decoration-break` utility that makes sense:

https://tailwindcss.com/docs/box-decoration-break

Hope it helps!

![image](https://user-images.githubusercontent.com/485747/133541995-6f0c05e4-3e34-4c4a-892c-c3e81d782a8b.png)

**Reply by simonswiss** [reply]
Also, as a side note, not sure if the code you shared was just demo stuff or what you use in your site, but: 

You should not put a `p` tag inside a `h1` – better to use a `span` here (and you can then skip the `inline` class since `span` is inline by default).
